src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -104,6 +111,24 @@ export class ViewConfigItemComponent implements OnChanges {
     }
   }
 
+  public setEditorConfig(type: string) {
+    const editorOptions = {
+      theme: this.selectedTheme,
+      mode: 'text/x-python',
+      lineNumbers: true,
+      lineWrapping: true,
+      foldGutter: true,
+      gutters: ['CodeMirror-linenumbers', 'CodeMirror-foldgutter', 'CodeMirror-lint-markers'],
+      autoCloseBrackets: true,
+      matchBrackets: true,
+      lint: true
+    };
+    if (type === 'JSON') {
+        editorOptions.mode = 'application/json';
```

Comment:
we should set completely new object to editorsOptions config? Instead of overriding the python mode only.

Reply:
```mode: 'application/json',
  gutters: ["CodeMirror-lint-markers"],
  styleActiveLine: true,
  lineNumbers: true,
  line: true,
  lint: true,```
May be just above with json-lint mod

Reply:
There is no need to set new object to editorsOptions config because all options are same for both cases, only mode is different.